nogotofail/mitm/connection/handlers/connection/log.py

```diff
@@ -27,7 +27,7 @@ class LoggingHandler(base.BaseConnectionHandler):
 
     FORMAT = (
         "[%(client_addr)s:%(client_port)s<=>%(server_addr)s:%(server_port)s "
-        "%(class)s](%(app_str)s) %(message)s")
+        "connection_id=\"%(connection_id)s\" %(class)s](%(app_str)s) %(message)s")
```

Comment:
This looks a bit different format wise than the rest of the message, maybe strip out the connection=\"<...>\"?